tests/e2e/api-core-tests/bin/wc-api-tests.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+#
+# Run package scripts
+#
+
+# Script help
+usage() {
+	echo 'usage: npx wc-api-tests <script>'
+	echo 'scripts:'
+	echo '         test - run API tests'
```

Comment:
`<group>` needs to be added to the usage help.

Reply:
Added!